### ISSUE TITLE
AddDefaultUI no longer needed

### ISSUE THREAD

**Issue by mrfixij** [aspnet/AspNetCore.Docs#15107]
I believe as of dotnet core 3.0's september release, this block is no longer valid.
services.AddDbContext&lt;ApplicationDbContext&gt;(options =&gt;
    options.UseSqlite(Configuration.GetConnectionString("DefaultConnection")));

services.AddDefaultIdentity&lt;ApplicationUser&gt;()
    .AddDefaultUI(UIFramework.Bootstrap4)
    .AddEntityFrameworkStores&lt;ApplicationDbContext&gt;();

I believe we should remove AddDefaultUi, since UIFramework is an internal enum.  Feel free to close this if I'm completely off base.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 21cb7455-3305-9316-5440-bdc22947a82d
* Version Independent ID: 137d4b94-7b26-3911-b22d-42c754a95fc1
* Content: [Introduction to authentication for Single Page Apps on ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/security/authentication/identity-api-authorization.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/identity-api-authorization.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @javiercn
* Microsoft Alias: **scaddie**

**Comment by javiercn** [comment]
Yep. `AddDefaultUI` doesn't offer an enum option anymore, the UIFrameworkVersion is now decided at build time in MSBuild.

**Comment by guardrex** [comment]
Thanks @mrfixij ... this is a dup of https://github.com/aspnet/AspNetCore.Docs/issues/15003.
